packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts

```diff
@@ -175,4 +185,8 @@ export class LoadBalancedFargateService extends cdk.Construct {
       });
     }
   }
+
+  private addLogDriver(prefix: string): AwsLogDriver {
```

Comment:
`addLogDriver` doesn't actually add anything to anything. It just returns a new logdriver.